src/index.ts

```diff
@@ -1,5 +1 @@
-import {Greeter} from './Greeter';
-
-const example = new Greeter('World');
-
-console.log(example.greet());
\ No newline at end of file
+export * from './Greeter';
```

Comment:
This an example of a ['barrel' index](https://basarat.gitbooks.io/typescript/docs/tips/barrel.html), where a library re-exports everything in a single index.